src/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/NativeMethods.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace System.DirectoryServices.ActiveDirectory
 {
+#pragma warning disable BCL0015 // CoreFxPort
```

Comment:
The complier was warning that the native calls may not be available on all platforms this library is available for.  Is there a better way to specify which platforms this is for so I don't need to disable this warning?